packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/template-literal-property/input.js

```diff
@@ -0,0 +1,10 @@
+const input = {};
+
+const {
+  given_name: givenName,
+  'last_name': lastName,
+  [`country`]: country,
+  [prefix + 'state']: state,
```

Comment:
Could you also add a test without this line, so that `allLiterals` will be true?

Reply:
Yes, I remove this `BinaryExpression`.

Reply:
Could you add this test, to test the last changes?